app/serializers/rest/instance_serializer.rb

```diff
@@ -59,7 +59,7 @@ def configuration
 
       statuses: {
         max_characters: StatusLengthValidator::MAX_CHARS,
-        max_media_attachments: 4,
+        max_media_attachments: MediaAttachment::MAX_MEDIA_ATTACHMENTS,
```

Comment:
The ruby side of this has been handled in https://github.com/mastodon/mastodon/pull/30433 - including exposing the value via api serializer change. This could rebase against that and be just (I think?) the JS-side changes?

Reply:
Okay, will update, didn't know there is another PR working on the exactly same thing.

Reply:
Cool, thanks -- and yes, not trying to make dupe PRs, but despite having comment on this in Feb, I forgot it existed and opened that other one while noticing the issue.

Reply:
done. could you please help with the workflow and merge? thanks

Reply:
leave it to you to resolve the conversation.

Reply:
@mjankowski could you please help to review it again and approve/merge if possible?

Reply:
I don't have merge permissions and I'm literally not qualified to review the JS portions of the change; but the spirit of things look correct to me and like an overall improvement, and I therefore wish you the absolute best of fates on your merge journey.

Reply:
thanks. @renchap could you please help with?